examples/templates/Example.js

```diff
@@ -0,0 +1,206 @@
+( function () {
+    'use strict';
+
+    var P = window.P;
+    var OSG = window.OSG;
+    var osg = OSG.osg;
+    var osgViewer = OSG.osgViewer;
+    var osgDB = OSG.osgDB;
+    var osgShader = OSG.osgShader;
+
+    var $ = window.$;
+
+
+    var Example = function () {};
+
+    Example.prototype = {
+        // user Config
+        // overriden by user param on url
+        // used for the GUI tweak
+        _config: {
```

Comment:
Is there a reason with vars are not initalized in the constructor instead of the prototype ?